src/epipeODE/__init__.py

```diff
@@ -0,0 +1,21 @@
+import dataclasses
+
+@dataclasses.dataclass
+class Point: # holds the spatial coordinates of a point
+    x: int
+    y: int
+    def dist(self, dest):
+        return ((self.x - dest.x) ** 2 + (self.y - dest.y) ** 2) ** 1/2
```

Comment:
```suggestion
        return ((self.x - dest.x) ** 2 + (self.y - dest.y) ** 2) ** (1/2)
```
just to make sure it is sqrt